.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # Test CODEOWNERS
 # Default owner(s)
-*	@tyamaguchi-ucla @yashpatel6 @maotian06 @sorelfitzgibbon @uclahs-cds/nextflow-wg
+*	@tyamaguchi-ucla @yashpatel6 @maotian06 @sorelfitzgibbon @ucla-cds/nextflow-wg
```

Comment:
I haven't been able to get rid of this codeowners error.  It doesn't like either of the above.

Reply:
The issue was that particular team didn't have any access for the current repo; I've added `uclahs-cds/nextflow-wg` to the repo so the file should pass with that as an owner now

Reply:
Got it, reset to `@uclahs-cds/nextflow-wg`